src/commands/generate/ckl_metadata.ts

```diff
@@ -1,99 +1,309 @@
-      assettype: enforceEnum('What is the asset type?', Object.values(Assettype)) || undefined,
-      // Resulting techarea options have typos. Yes, these typos really are how the enumerations are defined in STIG viewer's source code
-      techarea: enforceEnum('What is the tech area? ', Object.values(Techarea)) || undefined,
-      stigguid: enforceNonEmptyString('What is the STIG ID? ') || undefined,
-      targetkey: enforceNonEmptyString('What is the target key? ') || undefined,
-      webordatabase: String(enforceEnum('Is the target a web or database?', ['y', 'n']) === 'y'),
-      webdbsite: enforceNonEmptyString('What is the Web or DB site? ') || undefined,
-      webdbinstance: enforceNonEmptyString('What is the Web or DB instance? ') || undefined,
-      vulidmapping: enforceEnum('Use gid or id for vuln number?', ['gid', 'id']) || undefined,
+      }),
+      showCalendar: await confirm({message: 'Show the calendar?', default: true}),
     }
-    const validationResults = validateChecklistMetadata(cklMetadata as ChecklistMetadata)
-    if (validationResults.ok) {
-      fs.writeFileSync(flags.output, JSON.stringify(cklMetadata, null, 2))
-      console.log(`Checklist metadata file written at: ${path.resolve(flags.output)}`)
-    } else {
-      console.error(`Unable to generate checklist metadata:\n${validationResults.error.message}`)
+    profiles.push(profile)
+    addProfile = await confirm({message: 'Do you want to add another profile?', default: false})
+  }
+
+  interactiveValues.profiles = profiles
+  console.log('Profiles:', profiles)
+  console.log('interactiveValues:', interactiveValues)
+
+  // Collect the "assets" metadata information
+  const assets: {
+    marking: string
+    hostname: string
+    hostip: string
+    hostmac: string
+    hostfqdn: string
+    targetcomment: string
+    role: unknown
+    assettype: unknown
+    techarea: unknown
+    stigguid: string
+    targetkey: string
+    webordatabase: unknown
+    vulidmapping: unknown
+    webdbsite?: string // Added property
+    webdbinstance?: string // Added property
+  } = {
+    marking: await input({
+      message: 'What is the classification marking? [Unlcass, CUI, etc]:',
+    }),
+    hostname: await input({
+      message: 'What is the asset hostname?:',
+      default: hostname(),
+      validate(input: string) {
+        if (/^[\w.-]+$/.test(input)) { // skipcq: JS-0113
+          return true
+        }
+        return 'Please enter a valid hostname'
+      },
+    }),
+    hostip: await input({
+      message: 'What is the asset IP address?:',
+      validate(input: string) {
+        if (isEmpty(input) || /^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/.test(input)) { // skipcq: JS-0113
+          return true
+        }
+        return 'Please enter a valid IP address or leave blank'
+      },
+    }),
+    hostmac: await input({
+      message: 'What is the asset MAC address?:',
```

Comment:
I know we had a multiple MAC address issue before. Would this allow someone to put multiple MAC addresses? Would we need that here?